api/src/pcapi/scripts/reindex_music_offers/main.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import time
+import typing
+
+from pcapi.app import app
+from pcapi.core import search
+from pcapi.core.categories import subcategories_v2
+from pcapi.core.offers.models import Offer
+
+
+BATCH_SIZE = 1000
+
+logger = logging.getLogger(__name__)
+
+
+def get_offer_ids() -> typing.Generator[list, None, None]:
+    query = Offer.query.filter(
+        Offer.subcategoryId.in_(subcategories_v2.MUSIC_SUBCATEGORIES), Offer.isActive.is_(True)
+    ).with_entities(Offer.id)
+    yield from query.yield_per(BATCH_SIZE)
+
+
+def reindex_seance_cine_offers() -> None:
+    batch = []
+    start_time = time.time()
+    prev_time = start_time
+    for offer_id_row in get_offer_ids():
+        batch.append(offer_id_row[0])
```

Comment:
```suggestion
    for offer_id, in get_offer_ids():
        batch.append(offer_id)
```
le `row[0]` est un peu mystique, j'ai mis du temps à comprendre que ça venait de `query.with_entities`